fuuk/people/tests/test_views.py

```diff
@@ -437,7 +437,7 @@ def test_grants_coauthor(self):
         H3 = Human.objects.create(nickname='Person3_test')
         P3 = Person.objects.create(type='STAFF', first_name='Test', last_name='Coauthor', human=H3)
         # Prepare a grant with P3 as coauthor
-        Grant.objects.get(pk=1).co_authors.add(P3)
+        Grant.objects.first().co_authors.add(P3)
```

Comment:
We should keep the specific reference to the object. The underlying issue here is that the `Grant` objects created in `setUp` doesn't have static primary key which is expected by the test.

The correct solution would be to set the grant objects to instance variable and use that instead of a query.